src/lib/holocene/select/native-select.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  export let label = '';
+  export let id: string;
+  export let placeholder: string;
+  export let value = '';
+  export let options: { label: string; value: string }[] = [];
+</script>
+
+{#if label}
+  <label for={id} class={$$props.labelClass}>{label}</label>
+{/if}
+<select {value} {id} class={$$props.class} {placeholder} on:select>
```

Comment:
Probably bind the value so we can just do `<NativeSelect bind:value={} />` so we get reactivity on the value up and down.

Reply:
In the future maybe some default styling that works well enough to make it look like the non native counterpart for at least the initial view.